integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/adapters.py

```diff
@@ -102,6 +102,10 @@ class AnthropicClaudeAdapter(BedrockModelAdapter):
 
```

Comment:
I would add here a simple docstring, quickly describing the parameters in the kwargs that the class will read/use, e.g.:

```python
"""
model_kwargs: model configuration:
    - use_messages_api: Whether to use the messages API, default: True
    - include_thinking: Whether to include thinking output, default: True
    - thinking_tag: XML tag for thinking content, default: "thinking"
max_length: Maximum length of generated text
"""
```